core/embed/trezorhal/stm32u5/tamper.c

```diff
@@ -180,38 +229,8 @@ void tamper_init(void) {
 // Interrupt handle for all tamper events
 // It displays an error message
 void TAMP_IRQHandler(void) {
-  const char* reason = "UNKNOWN";
-
   uint32_t sr = TAMP->SR;
   TAMP->SCR = sr;
 
-  if (sr & TAMP_SR_TAMP1F) {
```

Comment:
Maybe we could use #ifndef BOARDLOADER to minimize the code size in the boardloader and leave full implementation in other binaries.

Reply:
see https://github.com/trezor/trezor-firmware/commit/b198c45bc41cf61217dde13438ca4a4d38aacc48